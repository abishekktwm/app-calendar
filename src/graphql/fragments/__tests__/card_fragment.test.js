import cardFragment from '../card_fragment';

describe('Card fragment', () => {
  it('match snapshot', () => {
    expect(cardFragment).toMatchSnapshot();
  });
});
