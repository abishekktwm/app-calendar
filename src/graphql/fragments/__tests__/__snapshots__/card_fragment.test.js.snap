// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card fragment match snapshot 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "kind": "FragmentDefinition",
      "name": Object {
        "kind": "Name",
        "value": "cardFragment",
      },
      "selectionSet": Object {
        "kind": "SelectionSet",
        "selections": Array [
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "id",
            },
            "selectionSet": undefined,
          },
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "title",
            },
            "selectionSet": undefined,
          },
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "due_date",
            },
            "selectionSet": undefined,
          },
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "suid",
            },
            "selectionSet": undefined,
          },
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "__typename",
            },
            "selectionSet": undefined,
          },
        ],
      },
      "typeCondition": Object {
        "kind": "NamedType",
        "name": Object {
          "kind": "Name",
          "value": "Card",
        },
      },
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 93,
    "start": 0,
  },
}
`;
